template/python3-http-armhf/index.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+from flask import Flask, request, jsonify
+from waitress import serve
+import os
+
+from function import handler
+
+app = Flask(__name__)
+
+class Event:
+    def __init__(self):
+        self.body = request.get_data()
+        self.headers = request.headers
+        self.method = request.method
+        self.query = request.args
+        self.path = request.path
```

Comment:
Please also ad request.files and request.form - they are needed for multipart/form-data requests. The `request.files` attribute is an [ImmutableMultiDict](https://tedboy.github.io/flask/generated/generated/werkzeug.MultiDict.html#werkzeug.MultiDict) of [FileStorage](https://tedboy.github.io/flask/generated/generated/werkzeug.FileStorage.html) (which is a file-ish ducktype), and form is an ImmutableMultiDict of strings which represent the textual multiparts which have no filename in their content-disposition. Maybe make a plain dictionary of lists from the MultiDict, to abstract away the Flask api.

Reply:
That won't be added at this time, but feel free to propose it as an issue.

Reply:
👍